src/core/Microsoft.PowerToys.Settings.UI.Runner/Microsoft.PowerToys.Settings.UI.Runner.csproj

```diff
@@ -31,6 +31,14 @@
     <OutputPath>..\..\..\$(Platform)\$(Configuration)\</OutputPath>
   </PropertyGroup>
 
+  <ItemGroup>
+    <None Remove="stylecop.json" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <AdditionalFiles Include="stylecop.json" />
```

Comment:
We should only reference the copy of stylecop from common instead of creating new one. Similar  example: https://github.com/microsoft/PowerToys/blob/master/src/modules/fancyzones/editor/FancyZonesEditor/FancyZonesEditor.csproj#L234